README.md

```diff
@@ -74,3 +74,67 @@ is as follows
 ## Merging into Main
 Once a PR is merged into the main branch the snapshot suffix will be deleted from the version.
 The assigned version number will be used to create a version release in artifactory and a Github tag for backup.
+
+
+## Using Filters
+
+### Overview
+JpaRest supports the use of filtering data during retrieval for certain get endpoints. These filters should be constructed in accordance with Spring Expression Language (SpEL) as they will then be converted into predicates used to query the database.
+
+### Supported Endpoints
+You can provide a filter as a request parameter for `GET` endpoints that return multiple resources. The filter expression should be assigned to a request parameter named `filter`.  
+Examples of supported endpoints for providing a filter parameter are as follows:
+- Getting multiple resources endpoint - `http://localhost:5000/resources/artists`
+- Getting multiple related resources for a parent resource - `http://localhost:5000/resources/concerts/37e813a2-bb28-11ec-8422-0242ac120001/artists`
+
+
+### Forming Expressions
+Filter expressions can be formed through the use of logical operators or SpEL method invocation. The expression requires an operator and 1 or 2 operands depending on the type of operator.
+
+#### Logical Operators
+When forming a filter expression using logical operators the following conditions must be met:
+- The left side operand must be a reference to a field on the entity. Literal values are not supported here.
+- The right side operand can be a field reference or a literal value, however the data type of the right  
+  side operand must match the data type of the left side operand.
+- The following operators are supported: `==, !=, >=, >, <=, <`
+- The operators can also be provided in the equivalent textual form such as: `eq, ne, ge, gt, le, lt`
+
+An example of the filter request parameter using a logical operator is as follows:
+`filter=id == 'dc27d5aa-4e7d-474e-98b6-ebf9aae9a471'`
```

Comment:
Is the entity name case sensitive?

Reply:
I've now updated the remaining areas to mention if they're case sensitive.